src/test/java/com/google/api/codegen/gapic/testdata/php/php_library.baseline

```diff
@@ -1724,6 +1734,13 @@ class LibraryServiceGapicClient
      */
     public function monologAboutBook(array $optionalArgs = [])
     {
+        $requestParams = new RequestParamsHeaderDescriptor([
+          'name' => $request->getName(),
+        ]);
+        $optionalArgs['headers'] = isset($optionalArgs['headers'])
+            ? array_merge($requestParams->getHeader(), $optionalArgs['headers'])
```

Comment:
Looks like this is broken because there is no `$requestParams` function parameter